www/src/components/layout.js

```diff
@@ -151,6 +152,7 @@ class DefaultLayout extends React.Component {
           <meta name="twitter:site" content="@gatsbyjs" />
           <meta name="og:type" content="website" />
           <meta name="og:site_name" content="GatsbyJS" />
+          <meta name="og:image" content={gatsbyIcon} />
```

Comment:
this will be relative path (something like `/static/some-hashed-name.png`) and I think og:image need absolute path right?

Reply:
You're totally right!